cmd/zed/validate.go

```diff
@@ -0,0 +1,204 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/jzelinskie/cobrautil"
+	"github.com/spf13/cobra"
+
+	v0 "github.com/authzed/authzed-go/proto/authzed/api/v0"
+	"github.com/authzed/spicedb/pkg/development"
+	"github.com/authzed/spicedb/pkg/tuple"
+	"github.com/authzed/spicedb/pkg/validationfile"
+	"github.com/authzed/spicedb/pkg/validationfile/blocks"
+	"github.com/charmbracelet/lipgloss"
+
+	"github.com/authzed/zed/internal/decode"
+)
+
+var (
+	success                = lipgloss.NewStyle().Bold(true).Foreground(lipgloss.Color("10")).Render("success!")
+	errorPrefix            = lipgloss.NewStyle().Bold(true).Foreground(lipgloss.Color("9")).Render("error: ")
+	errorMessageStyle      = lipgloss.NewStyle().Bold(true).Width(80)
+	linePrefixStyle        = lipgloss.NewStyle().Foreground(lipgloss.Color("19"))
```

Comment:
Can this just be the same as the regular code style ("8")? It's a bit distracting as blue. If you want to keep it blue, you can use "12".

Reply:
Used 12